server/src/Dockerfile

```diff
@@ -44,4 +44,6 @@ RUN set -ex \
     && rm -rf /root/.cache/pip
 
 # copy project
-COPY . /usr/src/app/
\ No newline at end of file
+COPY . /usr/src/app/
+RUN git clone https://github.com/pyronear/pyro-api.git
+RUN pip install -e pyro-api/client/.
```

Comment:
why not adding something like this `RUN pip install -r /usr/src/app/requirements.txt \` instead of those lines ?

Reply:
I've made the change and tested it, it works. You're right, it does look cleaner.